cdserver/db/seeds.rb

```diff
@@ -1,3 +1,45 @@
 User.create!(email: "sample_user@dhs.nyc.gov", password: "password")
 User.create!(email: "some_person@dhs.nyc.gov", password: "swordfish")
 User.create!(email: "anotherone@dhs.nyc.gov", password: "example")
+
```

Comment:
It shouldn't be possible to create a building without a shelter, since a shelter building cannot exist with out a shelter. what am i missing?